code/__DEFINES/power_defines.dm

```diff
@@ -40,3 +40,4 @@
 #define APC_FULLY_CHARGED 2
 
 #define KW *1000
+#define KJ *1000
```

Comment:
how does this matchup against similar defines in the BSH file?

Reply:
The BSH only has power defines, though they are just duplicates, come to think of it. I'll just move them here.